openapic/compiler.go

```diff
@@ -33,14 +34,33 @@ import (
 )
 
 type PluginCall struct {
-	Name   string
-	Output string
+	Name       string
+	Invocation string
 }
 
 func (pluginCall *PluginCall) perform(document *openapi_v2.Document, sourceName string) {
 	if pluginCall.Name != "" {
 		request := &plugins.PluginRequest{}
-		request.Parameter = ""
+
+		request.Invocation = pluginCall.Invocation
+		invocationParts := strings.Split(pluginCall.Invocation, ":")
+		var outputLocation string
+		switch len(invocationParts) {
+		case 1:
+			outputLocation = invocationParts[0]
+		case 2:
```

Comment:
Multiple options can be comma-separated, like this:

`openapic sample.json -myplugin_out=foo=123,bar=456:mydir`